lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -213,6 +213,9 @@ private StepResponse executeJourneyEvent(
                                         journeyChangeState.getInitialState()));
                 ipvSessionItem.setJourneyType(journeyChangeState.getJourneyType());
                 ipvSessionItem.setUserState(journeyChangeState.getInitialState());
+                ipvSessionItem.pushState(
+                        journeyChangeState.getJourneyType(), journeyChangeState.getInitialState());
```

Comment:
I think we should for completeness. It's also useful for keeping track of the users journey type. When we remove journey type from the session I think we'd lose it otherwise.

Reply:
Sure - when we go back we'll just need to know that it's ok to skip over it.